services/src/handlers/wfs.rs

```diff
@@ -185,25 +188,36 @@ async fn get_feature<C: Context>(
     // TODO: use correct session when WFS uses authenticated access
     let execution_context = ctx.execution_context(&Session::mock())?;
     let initialized = operator
+        .clone()
         .initialize(&execution_context)
         .context(error::Operator)?;
 
     // handle request and workflow crs matching
-    let workflow_spatial_ref = initialized.result_descriptor().spatial_reference();
-    let request_spatial_ref: SpatialReferenceOption = request.srs_name.into();
+    let workflow_spatial_ref: Option<SpatialReference> =
+        initialized.result_descriptor().spatial_reference().into();
+    let workflow_spatial_ref = workflow_spatial_ref.ok_or(error::Error::InvalidSpatialReference)?;
+
     // TODO: use a default spatial reference if it is not set?
-    snafu::ensure!(
-        request_spatial_ref.is_spatial_ref(),
-        error::InvalidSpatialReference
-    );
-    // TODO: inject projection Operator
-    snafu::ensure!(
-        workflow_spatial_ref == request_spatial_ref,
-        error::SpatialReferenceMissmatch {
-            found: request_spatial_ref,
-            expected: workflow_spatial_ref,
-        }
-    );
+    let request_spatial_ref: SpatialReference = request
+        .srs_name
+        .ok_or(error::Error::InvalidSpatialReference)?;
+
+    // perform reprojection if necessary
```

Comment:
we should add a todo to avoid initializing twice. However, i see that it is not trivial using the current implementation of the projection.

Reply:
b0e52e6416e7548ded6ed9f6ff1e5ec46d2c4909